pkg/deploy/generator/resources_dev.go

```diff
@@ -509,11 +509,11 @@ rm cron
 }
 
 const (
-	sharedKeyVaultName          = "concat(take(resourceGroup().name,15), '" + SharedKeyVaultNameSuffix + "')"
+	sharedKeyVaultName          = "concat(take(resourceGroup().name,10), '" + SharedKeyVaultNameSuffix + "')"
 	sharedDiskEncryptionSetName = "concat(resourceGroup().name, '" + SharedDiskEncryptionSetNameSuffix + "')"
 	sharedDiskEncryptionKeyName = "concat(resourceGroup().name, '-disk-encryption-key')"
-
-	SharedKeyVaultNameSuffix          = "-sharedKV"
+	// Conflicts with current development subscription. cannot have two keyvaults with same name
```

Comment:
Have you looked into moving the keyvault instead? 
https://docs.microsoft.com/en-us/azure/key-vault/general/move-subscription

Reply:
Thanks for the response @rolandmkunkel ! We cannot move it right now because that would impact our current workflow. If i move keyvault from current subscription, developers will be blocked from cluster creation

Reply:
How often do we expect to move it in future? I think it might be worth thinking about moving it instead, with a heads-up notice for the developers

Reply:
If i move the keyvaults today, developers cannot create clusters anymore. after looking at the portal, i noticed three dev clusters were created today. 

My attempt is to do a 0 impact migration, and switching to old subscription in case something goes wrong is a safeguard protecting it